### ISSUE TITLE
The following error occurs after listening

### ISSUE THREAD

**Issue by 0x7e-1sq** [HavocFramework/Havoc#151]
qt.qpa.xcb: QXcbConnection: XCB error: 3 (BadWindow), sequence: 4959, resource id: 11165776, major code: 40 (TranslateCoords), minor code: 0

help me 
thanks..

**Comment by 0x7e-1sq** [comment]
And I was unable to connect to a Havoc server through the EXE that Havoc generated

**Comment by Cracked5pider** [comment]
can you provide more information? Like screenshots? Thanks for reporting

**Comment by 0x7e-1sq** [comment]
![image](https://user-images.githubusercontent.com/57382546/194759249-dbf69272-93fa-4160-89e4-a8dbfddc6de2.png)

**Comment by Cracked5pider** [comment]
This is not a listener error. this is simply an XCB log. Does this crashes the client tho? Or what is your issue ?

**Comment by 0x7e-1sq** [comment]
Will this error affect my use of Havoc? Because I found that no one else has this error, and there was no feedback from the Havoc server after I ran the EXE generated by Havoc.

**Comment by Cracked5pider** [comment]
This won't affect the use of Havoc. Did your client crash? Did something else unexpected happen? What version are you on (please provide the commit hash you are on)

**Comment by 0x7e-1sq** [comment]
No, everything is fine with my client, BUT there is no feedback from the server after the generated EXE runs.
It has been bothering me for a day. I tried to generate exe dll shellcode, but the server did not get any feedback.
![image](https://user-images.githubusercontent.com/57382546/194759885-75a54858-4440-4539-ad25-02fa84fe4a27.png)
Havoc is latest

My feedback here is that I can't listen to any conversations
![image](https://user-images.githubusercontent.com/57382546/194760073-331b9281-381b-49bd-a88b-2c72d89c779b.png)

**Comment by Cracked5pider** [comment]
What version are you on? 
can you type `git show -s` in the Havoc directory ?

**Comment by 0x7e-1sq** [comment]
![image](https://user-images.githubusercontent.com/57382546/194760701-499edc9d-cfda-4c77-880c-607e596ad244.png)

commit 2ef6cf86b8cf85a8ce031846c2e7d57a020a551a (HEAD -> main, origin/main, origin/HEAD)
Merge: 7052442 c5d66e9
Author: C5pider <51360176+Cracked5pider@users.noreply.github.com>
Date:   Sun Oct 9 00:57:44 2022 +0200

    Merge pull request #149 from HavocFramework/dev
    
    marget dev branch into main

**Comment by Cracked5pider** [comment]
Alright can you share with me the running listeners and how you generate a paylaod?

**Comment by 0x7e-1sq** [comment]
![image](https://user-images.githubusercontent.com/57382546/194760912-6d590a30-8464-46f3-858d-2b2f0f9ea545.png)
![image](https://user-images.githubusercontent.com/57382546/194760992-80805cc6-3e9f-4953-b3fe-bfd84e753f12.png)
![image](https://user-images.githubusercontent.com/57382546/194761012-c16d3ea0-284c-4721-a1b0-c9196ab08b2c.png)
![image](https://user-images.githubusercontent.com/57382546/194761068-e01c2def-9ce0-4d5f-90ad-fb3c495f7ac1.png)
![image](https://user-images.githubusercontent.com/57382546/194761195-065679bc-627d-4851-9d1b-9ba7aa9659e3.png)
![image](https://user-images.githubusercontent.com/57382546/194761243-d3fb6d3a-3ec6-4ae4-822f-9e07484b497e.png)

That's how I do it
I don't think headers and uris are necessary? I also tried to fill in these two options according to github before, but still failed.

**Comment by Cracked5pider** [comment]
The uris are not necessary (if not specified the `/` is going to be used as the default)but I think it has something to do with the fact that your system is Chinese. There is currently no real support for Unicode environments. 
But can you restart the teamserver with the flag `--debug-dev` and recompile the agent and share with me the Debug prints that the agent gives away? Thanks that would help me a lot to support Unicode hosts.

**Comment by 0x7e-1sq** [comment]
Do you mean that I need to recompile teamserver?

What do I need to do?

Add parameters when compiling

Again, add parameters at boot time

**Comment by Cracked5pider** [comment]
start the teamserver with the following flag `--debug-dev`. This flag tells the teamserver to compile all agents in debug mode. which allows me to detect bugs/errors and other issues. 
For example: `./teamserver server --profile profiles/havoc.yaotl -v --debug-dev` 
After you did this you can recompile the agent in the payload generator. After running it you will get a console log with debug messages. Can you share then the entire debug log with me ? thx.

**Comment by 0x7e-1sq** [comment]
![image](https://user-images.githubusercontent.com/57382546/194762148-97f5193d-34d8-4149-80fc-bd7d425d71ff.png)

All I found was this log
![image](https://user-images.githubusercontent.com/57382546/194762139-052b0ae9-1e28-436c-95e1-f97287bc5e7c.png)

![image](https://user-images.githubusercontent.com/57382546/194762227-1e662b78-873b-442c-8a7b-825b536e0449.png)

**Comment by Cracked5pider** [comment]
No what i have meant is that you should regenerate a payload and execute it. The demon payload is going to show a console with debug prints.

**Comment by 0x7e-1sq** [comment]
![image](https://user-images.githubusercontent.com/57382546/194762542-84e3c54d-b778-4ea6-b4e4-07fed37f9c6a.png)
![image](https://user-images.githubusercontent.com/57382546/194762624-e2a3b6d1-03b6-4ce1-a825-92cde5fc6c1a.png)
And then you just keep repeating falied 122 
Is that right?

**Comment by Cracked5pider** [comment]
Alright, perfect thx. Going to investigate this issue.

**Comment by 0x7e-1sq** [comment]
Thank you for looking forward to your masterpiece
